packages/css-framework/src/components/_phase-banner.scss

```diff
@@ -0,0 +1,20 @@
+.rn-phase-banner {
+  background-color: color(primary, 100);
+  padding: spacing(2) 0;
+}
+
+.rn-phase-banner__text {
+  display: inline-block;
+  margin-left: spacing(1);
+  font-size: font-size(xs);
+}
+
+.rn-phase-banner__text a {
```

Comment:
do we need to style the 'a' or can it be a class?

Reply:
We need to style the a really. It's potentially user generated content.